unit_tests/sources/declarative/auth/test_oauth.py

```diff
@@ -129,6 +129,25 @@ def test_refresh_without_refresh_token(self):
         }
         assert body == expected
 
+    def test_get_auth_header_without_refresh_token_and_without_refresh_token_endpoint(self):
+        """
+        Coverred the case when the `access_token_value` is supplied,
+        without `token_refresh_endpoint` or `refresh_token` provided.
+
+        In this case, it's expected to have the `access_token_value` provided to return the permanent `auth header`,
+        contains the authentication.
+        """
+        oauth = DeclarativeOauth2Authenticator(
+            access_token_value="my_access_token_value",
```

Comment:
❓ Why do `access_token_value="my_access_token_value",` and `client_secret="{{ config['client_secret'] }}",` differ in their semantics?

I assumed access_token (like refresh token, client id, and client secret) would be retrieved from the config.

e.g. 
```
access_token_value ="{{ config['access_token'] }}",
```

Reply:
Will correct this one shortly.